src/desktop/package.json

```diff
@@ -139,7 +139,7 @@
     "electron-updater": "^3.0.3",
     "entangled-node": "rihardsgravis/entangled-node",
     "i18next": "^11.5.0",
-    "iota.lib.js": "^0.4.7",
+    "iota.lib.js": "^0.5.0",
```

Comment:
Ideally, whatever is using `iota.lib.js` (looks like `Electron.js`) should import it from `shared` to avoid installing duplicate dependencies